wallet-new/src/Cardano/Wallet/Kernel/DB/Spec/Update.hs

```diff
@@ -98,14 +98,17 @@ applyBlock (prefBlock, _bMeta) checkpoints
     where
         utxo'        = checkpoints ^. currentUtxo
         utxoBalance' = checkpoints ^. currentUtxoBalance
-        pending'     = checkpoints ^. currentPendingTxs
 
-        (utxo'', balance'') = updateUtxo prefBlock (utxo', utxoBalance')
-        pending''           = updatePending prefBlock pending'
-        -- TODO(@uroboros/ryan) applyBlock.updateExpected/updateBlockMeta
+        (utxo'', balance'') = updateUtxo      prefBlock (utxo', utxoBalance')
+        pending''           = updatePending   prefBlock (checkpoints ^. currentPendingTxs)
+        blockMeta''         = updateBlockMeta prefBlock (checkpoints ^. currentBlockMeta)
+        -- TODO(@uroboros/ryan) applyBlock.updateExpected
         -- (as part of CBR-150 Extend pure data layer to support rollback)
         expected''          = checkpoints ^. currentExpected
-        blockMeta''         = checkpoints ^. currentBlockMeta
+
+updateBlockMeta :: PrefilteredBlock -> BlockMeta -> BlockMeta
+updateBlockMeta PrefilteredBlock{..} meta
+    = pfbMeta `mappend` meta
```

Comment:
It probably doesn't really matter, but the order of the operands to `mappend` here makes me a little uneasy.